paper/paper.md

```diff
@@ -18,119 +18,121 @@ authors:
+Facilitating the transition from continuum to fragmentation becomes straightforward by representing each material with its own set of particles, effectively addressing material interface issues.
+This also simplifies the coupling of different singlephysics into a multiphysics system.
 
-There are several open-source software projects specialized for SPH methods, including [@DualSPHysics], [@SplisHSPlasH] and [@SPHinXsys],
-written in C++, and  [@PySPH], written in Python. These frameworks emphasize performance and are designed for a variety of SPH schemes.
+There are several open-source software projects specialized for SPH methods, including DualSPHysics [@Dominguez:2021], SPlisHSPlasH [@Bender] and SPHinXsys [@Zhang:2021],
+written in C++, and  PySPH [@Ramachandran:2021], written in Python.
+These softwares reflect the advantages of the SPH methods by simulating problems such as fluid-structure interaction (FSI) and free surfaces [@O_Connor:2021] or complex geometries [@Laha:2024].
 
-TrixiParticles.jl provides support for developing or testing new SPH methods and also for simulating or coupling other particle-based methods such as DEM.
-Another aspect is that TrixiParticles.jl is not configured at runtime via a parameter file, as it is the case with other numerical simulation codes.
-Instead, each simulation is configured and set up by pure Julia code.
-This makes it easy for users to add custom functionality even without touching the source code.
-The latter is further enhanced by its seamless support for GPU acceleration, integrating advanced computational capabilities without necessitating separate codebases.
+TrixiParticles.jl is written in Julia and thus combines the advantage of easy and rapid prototyping with the ability of high performance computing using multicore parallelization and hardware accelerators.
+The software package provides support for developing or testing new SPH methods and also for simulating or coupling other particle-based methods such as DEM.
+Another aspect is the ability to add custom methods or particle interactions without modifying the source code, since each simulation is configured and set up by pure Julia code.
 
-# Scientific background
 
-In TrixiParticles.jl, particles of a single particle-based method are grouped into a so called \emph{system}.
+# Overview of particle-based simulation
+
+In TrixiParticles.jl, particles of a single particle-based method are grouped into what we refer to as a \emph{system}.
 The interaction between two particles is defined by the types of their systems. This approach makes it easy to add new methods and different physics.
 
-![Particles of two different systems in a simulation domain. \label{fig:systems}](systems.png){width=40%}
+![Particles of two different systems in a simulation domain. The black and gray dashed circle represent the search radius for neighboring particles of particle $a$ and $b$, respectively.\label{fig:systems}](systems.png){width=40%}
```

Comment:
```suggestion
![Particles of two different systems in a simulation domain. The black and gray dashed circles represent the search radii for neighboring particles of particles $a$ and $b$, respectively.\label{fig:systems}](systems.png){width=40%}
```
Also, can you please change the symbols in the image accordingly to `a` and `b` without the superscripts?